nx-workspace/package.json

```diff
@@ -49,6 +49,7 @@
     "react": "16.13.1",
     "react-dom": "16.13.1",
     "react-router-dom": "^5.2.0",
+    "redux-saga": "^1.1.3",
```

Comment:
Hm - is it just me getting a compilation error about react-redux missing? (in particular, `web/src/components/templates/Login.tsx` is looking for it)
Looking at the history it looks like some redux things got removed when master was merged in earlier today?

Reply:
good catch, thanks! I have accidentally installed it globally and caught myself out.
Added now. a8f545118